src/test/README.md

```diff
@@ -0,0 +1,142 @@
+# How To Test C++ And JS
+
+Usually to test against Javascript Core (JSC) we have to boot up an Android emulator, with all its dependencies (SDK, NDK, etc.) and then
+go through the suffering of having to create/load a react-native dummy app. Usually this way of working will require at between `20-40G` of space and lots of patience.
+
+To avoid all that pain we should target JSC directly and avoid the expensive middlemen for all code that aims directly to JSC (roughly 90%) and while 
```

Comment:
```suggestion
To avoid all that complexity we should target JSC directly and avoid the expensive middlemen for all code that aims directly to JSC (roughly 90%) and while 
```

Reply:
Isn't this something that your implementation already does?  ("should" makes it sound like we've taken a shortcut in our implementation that we shouldn't have)

Reply:
Just thought that this addition (Catch3 and the JSC boilerplate) might deserve a justification as to why I think is good to have it.